src/sentry/monitors/models.py

```diff
@@ -687,7 +695,12 @@ class MonitorIncident(Model):
     This represents the final OK check-in that we receive
     """
 
-    grouphash = models.CharField(max_length=32)
+    grouphash = models.CharField(max_length=32, default=default_grouphash)
+    """
+    Used for issue occurances generation. Failed check-ins produce occurance
```

Comment:
```suggestion
    Used for issue occurrences generation. Failed check-ins produce an occurrence
```

Reply:
will cleanup in a follow up, wanna get this in before it conflicts